src/cluster.c

```diff
@@ -2137,23 +2124,23 @@ static int prepareCommand(valkeyClusterContext *cc, struct cmd *command) {
     return VALKEY_OK;
 }
 
-int valkeyClusterSetConnectCallback(valkeyClusterContext *cc,
-                                    void(fn)(const valkeyContext *c,
-                                             int status)) {
-    if (cc->on_connect == NULL) {
-        cc->on_connect = fn;
+int valkeyClusterOptionsSetConnectCallback(valkeyClusterOptions *options,
+                                           void(fn)(const valkeyContext *c,
+                                                    int status)) {
+    if (options->connect_callback == NULL) {
+        options->connect_callback = fn;
         return VALKEY_OK;
     }
     return VALKEY_ERR;
 }
```

Comment:
Why is this kept? Can't the users just set this directly on the options object too?